src/large_scale_condensation.jl

```diff
@@ -0,0 +1,47 @@
+function get_large_scale_condensation_tendencies!(
+    Diag::DiagnosticVariables{NF},
+    M::ModelSetup{NF},
+) where {NF<:AbstractFloat}
+    @unpack pres_ref, cp, alhc, alhs, gravity, τ, RH¹, ΔRH, rhb = M.constants  # Let's give some of these constants better names?
+    @unpack nlon, nlat, nlev, σ_levels_half, σ_levels_full, σ_levels_thick =
+        M.geospectral.geometry
+    @unpack temp_grid, humid_grid, pres_surf_grid, cloud_top, precipitation_ls = Diag.grid_variables
+    @unpack temp_tend, humid_tend = Diag.parametrization_tendencies
+    @unpack cloud_top = Diag.intermediate_variables
+
+    # Use mathematical notation for consistency with the documentation
```

Comment:
I see that motivation, and I agree that we may want that for the beginning to make sure everything works well, otherwise I'd vote to break with the mathematical notation. I already struggle to remember what `cp, alhc, alhs, τ, rhb` etc stand for, hence we started using `gravity` instead of `g` etc. But let's discuss that!